server/sonar-web/src/main/js/apps/overview/meta/MetaTagsSelector.js

```diff
@@ -68,11 +66,11 @@ class ProjectTagsSelectorContainer extends React.PureComponent {
   };
 
   onSelect = (tag /*: string */) => {
-    this.props.setProjectTags(this.props.project, [...this.props.selectedTags, tag]);
+    this.props.setProjectTags([...this.props.selectedTags, tag]);
   };
 
   onUnselect = (tag /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)